### DISCUSSION TITLE
Non square thumbnails

### DISCUSSION THREAD

**Discussion by SolemnSpirit** [thumbsup/thumbsup#255] [Q&A] [answered]
Is there any way to generate thumbnails that are just reduced size versions of the original image (say 50% smaller) rather than making them square ?

**Comment by rprieto** [comment, marked as answer]
Hi. Thumbsup generates a few resized images: small square thumb, small proportional, and large proportional.

Many themes use the square thumb in "grid" mode but you don't have to. Some themes (e.g. flow) use the small image instead.

You can look at the source code to create your own theme too. The only difference is using `urlss.small` instead of `urls.thumb` in the image tag.
